### ISSUE TITLE
Button overlapping copy

### ISSUE THREAD

**Issue by bmartinez287** [UTCWeb/particle#215]
https://new.utc.edu/enrollment-management-and-student-affairs/cohort-program
<img width="855" alt="Screen Shot 2021-04-30 at 2 19 40 PM" src="https://user-images.githubusercontent.com/39039024/116737371-27696b80-a9bf-11eb-8db0-91a75ac156dd.png">

**Comment by bmartinez287** [comment]
Reported by April Cox https://utcwebdev.atlassian.net/browse/UWR-2412

**Comment by bmartinez287** [comment]
Not sure why she had to use that much text. It's something to look into whether is fixable or how to reduce this type of issue.
Also, she said this 
"I'm building a page for the Cohort program and have a section with three cards. The center card has a button link. It's overlapping the paragraph copy. I fixed this by added a paragraph return."
This fix is a tricky one. Mostly due CKEditor allowing large amounts of text.

**Comment by brittrich** [comment]
Thanks for reporting, Bernardo, I'll take a look and see what the fix would be.

**Comment by brittrich** [comment]
So, I did look at this a bit...and I think, really, there is some reconfiguring that needs to happen on the cards. I think there is a much simpler way to accomplish the button alignment using flex (I used some older-school methods here) and I could also move the cards to use tailwind at the same time. That being said, I know it isn't the right time to overhaul a component that is so heavily used on the site.

I wonder whether @UTCbridget might have time to take a look at the card block with me post-migration and we could identify any needed improvements that to implement if we wanted to take a look at it and revise this to use tailwind. 

A few possible items I could see:

-margin corrections (with button alignment and with the general card alignment when they are used under paragraphs)
-less shadow?
-Change from Exo titles...if we were so bold...we could use a semibold large version of Open Sans instead?
-Other things I'm not thinking of or anything that would align better with future design directions?

I know there is a lot going on right now to meet deadlines, so no rush to weigh in on this, just putting it out there as a future task perhaps.
